src/main/java/org/carlmontrobotics/subsystems/IntakeShooter.java

```diff
@@ -72,6 +72,7 @@ public IntakeShooter() {
         outtakeDistanceSensor.setRangingMode(RangingMode.Short, 24);
         outtakeMotorVortex.setSmartCurrentLimit(60);
         SmartDashboard.putNumber("Intake target RPM", 0);
+        SmartDashboard.putNumber("Vortex volts", 0);
```

Comment:
comment out unused smartdashboard values to avoid clutter if you aren't going to test anymore